BlockServer/core/config_list_manager.py

```diff
@@ -162,18 +177,16 @@ def _update_component_dependencies_pv(self, name):
             configs = self._comp_dependecncies[name]
         if name in self._component_metas.keys():
             # Check just in case component failed to load
-            pv_name = BlockserverPVNames.get_dependencies_pv(self._component_metas[name].pv)
-            self._ca_server.updatePV(pv_name, compress_and_hex(json.dumps(configs)))
+            self._update_pv_value(self._component_metas[name].pv + DEPENDENCIES_PV,
+                                  compress_and_hex(json.dumps(configs)))
```

Comment:
This doesn't look right. Surely it should be using the names from the block server PV names class